SSA/Core/WellTypedFramework.lean

```diff
@@ -66,6 +97,36 @@ def trdTriple [Goedel β] {k₁ k₂ k₃ : UserType β} : ⟦k₁.triple k₂ k
 end UserType
 
 
+namespace Example
+
+inductive UserType (β : Type) : Type where
```

Comment:
I'm not sure I understand the point of this. Is it testing code you forgot to remove?